### DISCUSSION TITLE
Problem sending invite mails from the workspace

### DISCUSSION THREAD

**Discussion by Kulturserver** [OpnForm/OpnForm#587] [Q&A] [answered]
Hello, we've installed the latest version of opnform on our server and everything is working fine, except for sending the invite mails to the workspace. The UI tells us the mail has been sent, but it actually isn't, it doesn't reach the mailservers we checked. Is there a configuration that might be missing? Sending mails from the forms works perfectly, just the invite mails in the workspace don't. 
Any help would be highly appreciated - thanks!

Edit: here's what we entered in the docker-compose.yml - not sure, if it's correct this way:

> MAIL_TYPE: 'smtp'
>       SMTP_SERVER: 'mail.culturebase.org'
>       SMTP_USERNAME: 'xxxxx@kulturserver.de'
>       SMTP_PASSWORD: 'xxxxxxxxx'
>       SMTP_USE_TLS: 'true'        
>       SMTP_PORT: 587

we've already checked with setting use_tls to false, which didn't make a difference.

**Comment by JhumanJ** [comment]
Hi there, any logs to share?Le 7 oct. 2024 à 12:12, Alex Klotz ***@***.***> a écrit :﻿
Hello, we've installed the latest version of opnform on our server and everything is working fine, except for sending the invite mails to the workspace. The UI tells us the mail has been sent, but it actually isn't, it doesn't reach the mailservers we checked. Is there a configuration that might be missing? Sending mails from the forms works perfectly, just the invite mails in the workspace don't.
Any help would be highly appreciated - thanks!

**Reply by Kulturserver** [reply]
Sure - here is what we found, it looks like everything's ok, but the mail never got received by any of the mailservers we tested...
[docker-error.log](https://github.com/user-attachments/files/17289331/docker-error.log)

**Comment by JhumanJ** [comment, marked as answer]
Where exactly did you setup these env variables? Please make sure to be editing `api/.env` - more info here: https://docs.opnform.com/configuration/email-setup

**Comment by Kulturserver** [comment]
Thanks a lot, this solved our problem!
